### ISSUE TITLE
BBReply?

### ISSUE THREAD

**Issue by smreyno** [Arx-Game/arxcode#87]
Is it possible to implement some kind of reply feature on certain parts of the bboard? Similar to what exists now for Proclamations, actually, but probably allowing people to be more verbose. I'm thinking particularly of Systems Discussion. If the goal is for that to become a place to discuss ideas/feedback, it'd be nice if you could reply to a post about a particular issue/idea, rather than creating entirely new posts. In the past, I've felt like discussion that happened there was disconnected and hard to follow due to each post being separate. It's not an ideal forum for back-and-forth, imo, but maybe it could be with some minor changes (if these are minor).

**Comment by TehomCD** [comment]
Not really feasible with the current bulletin boards, but converting to something like Paxboards would be a great way to implement it. The current boards have some pretty nasty limitations due to their implementation- some of the lag spikes you might see in game are probably a result of someone checking unread posts after a server restart.

**Comment by smreyno** [comment]
Yeah, that makes sense. I don't think it honestly makes a ton of sense to try and channel interactive discussion to the current bboards. It's at least not something I'd want to interact with as a player and it would likely make me less apt to participate if it was the primary forum for such things, though maybe some people don't mind idk. The promise of something more web based would be sweet, though!

**Comment by rozzingit** [comment]
PAXBOARDS PAXBOARDS
